frontend/src/commons/SoccerTeam.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+export default function SoccerTeam({ soccerTeam }) {
+  return <SoccerTeamStyled>{soccerTeam.name}</SoccerTeamStyled>;
+}
+
+const SoccerTeamStyled = styled.span`
```

Comment:
I feel span is not the right element type for this component, how about a `section`?